app/src/main/java/com/infomaniak/drive/data/cache/FileController.kt

```diff
@@ -466,6 +467,7 @@ object FileController {
                 }
             }
         }
+        deleteDriveIntoDriveInfo(userId)
```

Comment:
Here remove the existing import and write it like this : 
```suggestion
        DriveInfosController.deleteDrive(userId)
```